ray-operator/main.go

```diff
@@ -167,6 +168,12 @@ func main() {
 	exitOnError(ray.NewRayJobReconciler(mgr).SetupWithManager(mgr),
 		"unable to create controller", "controller", "RayJob")
 
+	if os.Getenv("ENABLE_WEBHOOKS") == "true" {
+		if err = (&rayv1.RayCluster{}).SetupWebhookWithManager(mgr); err != nil {
+			setupLog.Error(err, "unable to create webhook", "webhook", "RayCluster")
+			os.Exit(1)
+		}
+	}
```

Comment:
adapted from kubebuilder docs

- https://book.kubebuilder.io/cronjob-tutorial/running.html?highlight=ENABLE_WEBHOOKS#optional
- https://book.kubebuilder.io/cronjob-tutorial/main-revisited.html?highlight=ENABLE_WEBHOOKS#you-said-something-about-main